opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -256,7 +256,7 @@ def end(self, end_time: Optional[int] = None) -> None:
             if not self.is_recording_events():
                 return
             if self.start_time is None:
-                raise RuntimeError("Calling end() on a not started span.")
+                raise RuntimeError("Calling end() on an ended span.")
```

Comment:
Ops, is this correct? I don't see why this log message should be changed. Semantically we don't know here if span has ended and just checking if span started.

Reply:
Yep, you are right. Will revert the change.